ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -1063,5 +1064,10 @@ export default defineComponent({
     margin-left: 20px !important;
     padding-left: 20px !important;
   }
+
+.horizontal-divider {
+  border-color: $gray7;
+  max-width: 4px;
```

Comment:
Doesn't seem very wide, is this being overridden or is the divider meant to be 4px wide?